activesupport/lib/active_support/core_ext/array/grouping.rb

```diff
@@ -113,4 +113,15 @@ def split(value = nil)
       results
     end
   end
+
+  # Returns a copy of the Array without the specified elements.
+  #
+  #   people = ["David", "Rafael", "Aaron", "Todd"]
+  #   people.without "Aaron", "Todd"
+  #     => ["David", "Rafael"]
+  #
+  # Note: This is an optimization of `Enumerable#without`.
```

Comment:
`Note: This is an optimization of `Enumerable#without` that uses Array#- instead of Enumerable#reject for performance reasons.`